packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -41,7 +49,12 @@ export const DefaultCell = (props: TableCellProps) => {
   }
 
   return (
-    <div {...cellProps} className={cellStyle}>
+    <div
+      {...cellProps}
+      onMouseEnter={showActions ? onMouseEnter : undefined}
+      onMouseLeave={showActions ? onMouseLeave : undefined}
+      className={cellStyle}
+    >
       {!hasLinks && <div className={tableStyles.cellText}>{value}</div>}
```

Comment:
We also tried to remove this div as well, but there are some viz within table cell that require the cell to be display:flex that won't work with text-overflow:ellipsis

Reply:
Addressed in https://github.com/grafana/grafana/pull/76916